### PR TITLE
the pr fixing my previous pr

### PR DESCRIPTION
**Pull request by birdlover32767**
this will be a draft until im sure everything is fixed

### changes ###
* `displayDialougeLine` and `displayDialougeLine` are **deprecated** in favor of `displayDialogueLine` and `displayGlobalDialogueLine` respectively
  note: **this is only a name change**, but please use the correctly spelled `displayDialogueLine`
* meteorites don't affect protected areas *unless* you add `.meteorite` to the protection
* meteoric nodes have slightly different drops
* meteoric antimatter has the meteoric overlay inverted now (the red is blue now)
* attractors/repulsors increase/decrease meteorite spawns 4x less often
* antimatter-matter annihilation produces less particles
* phosphor now connects to other phosphor visually like emittrium reactor shells
* new anti-annihilator and compressed core dust quest
* more quests use the `[[text]]` format instead of `"text"` because `\n`
* changed some quest text
* compressed core dust doesn't annihilate in contact with antimatter anymore
* added the antimatter meteorite trail texture because i forgot to do that for some reason
* antimatter dust can be turned back into matter dust
* compressed core dust def has been moved into nodes.lua

### CODE REVIEW

**Comment by TheEt1234** [comment]
also you should make attractors not add to the global spawn rate, and also make explosions respect protections

**Comment by birdlover32767** [comment]
alr

**Comment by TheEt1234** [comment]
@birdlover32767 should zander merge this

File: mods/sbz_meteorites/mod.conf [hunk 1]
```diff
@@ -1,2 +1,2 @@
 name = sbz_meteorites
-depends = sbz_base, sbz_resources, sbz_power
\ No newline at end of file
+depends = sbz_base, sbz_resources, sbz_power, areas
```
Comment (TheEt1234) [inline mods/sbz_meteorites/mod.conf:2]: i don't think having minetest.is_protected requires requiring areas but sure

File: mods/sbz_progression/init.lua [hunk 2]
```diff
@@ -4,28 +4,23 @@ local modpath = minetest.get_modpath("sbz_progression")
...
 
-function displayGlobalDialougeLine(text)
+function displayGlobalDialogueLine(text)
     minetest.chat_send_all("⌠ " .. text .. " ⌡")
     minetest.sound_play("dialouge", {
         gain = 1.0,
     })
 end
 
-minetest.register_chatcommand("cheat_hacker", {
```
Comment (TheEt1234) [inline mods/sbz_progression/init.lua:22]: wait there used to be an achievment asociated with this command, is it still in the game

**Review by TheEt1234 — APPROVED** [review]

File: mods/sbz_progression/init.lua (hunk 2 above)
Comment (birdlover32767) [reply mods/sbz_progression/init.lua:22]: no, i dont think so

File: mods/sbz_meteorites/mod.conf (hunk 1 above)
Comment (birdlover32767) [reply mods/sbz_meteorites/mod.conf:2]: i mean all of the checks happen after load time but... yea

**Comment by birdlover32767** [comment]
probably

**Comment by TheEt1234** [comment]
@ChefZander then maybe merge(?)

**Pull request merged**